.github/actions/setup-build/action.yml

```diff
@@ -57,11 +57,9 @@ runs:
     - if: inputs.shorten-hls == 'true'
       name: Workaround shorten binary names
       run: |
-        sed -i.bak -e 's/haskell-language-server/hls/g' \
-                    -e 's/haskell_language_server/hls/g' \
+        sed -i.bak -e 's/executable haskell-language-server/executable hls/g' \
```

Comment:
This now _just_ shortens the executable names, _not_ the package name. That makes it easier to refer to the package.